.flake8

```
@@ -0,0 +1,28 @@
+[flake8]
+filename =
+    ./scripts/*.py,
+    ./src/*.py,
```

Linting of tests won't work without this:
```suggestion
filename =
    ./scripts/*.py,
    ./src/*.py,
    ./tests/*.py
```